lsp-clients/n4js-vscode-extension/src/ExtensionProvider.n4js

```diff
@@ -93,19 +93,37 @@ export function getActivate(vscode: any+, vscodeLC: any+): any {
         // Create the language client and start the client.
         let lc = new vscodeLC.LanguageClient(CHANNEL_NAME, serverOptions, clientOptions, true);
         lc.onReady().then(() => {
-            const requestType = new vscodeLC.RequestType ('n4/documentContents');
+
+            
+            const rtDocumentContents = new vscodeLC.RequestType('n4js/documentContents');
             const textDocumentContentProvider = {
                 provideTextDocumentContent: (uri: any+, token: any+): any+ => {
-                    return lc.sendRequest(requestType, { uri: uri.toString() }, token).then((v: string): string => {
+                    return lc.sendRequest(rtDocumentContents, { uri: uri.toString() }, token).then((v: string): string => {
                         return v || '';
                     });
                 }
             };
             context.subscriptions.push(vscode.workspace.registerTextDocumentContentProvider('n4scheme', textDocumentContentProvider));
+            
+            const setVerboseLevelRT = new vscodeLC.RequestType('debug/setVerboseLevel');
```

Comment:
Maybe use `n4js/debug/..` as the request path?

Reply:
As discussed: We move the feature to the general feature set of Xtext LSP.